### DISCUSSION TITLE
coffee-editor依赖的modelserver包和github上的modelserver项目代码差异大？

### DISCUSSION THREAD

**Discussion by 136861535** [eclipse-emfcloud/emfcloud#136] [Coffee Editor]
您好，coffee-editor依赖的modelserver包和github上的modelserver项目代码差异大？我们应该以哪个为准呢，我们应该着重研究哪一个呢？

**Comment by eneufeld** [comment]
Hi @136861535 ,
the coffee editor is an example application build on different technologies including the modelserver project (https://github.com/eclipse-emfcloud/emfcloud-modelserver) . 
You can find more details here: https://www.eclipse.org/emfcloud/

Also please post your questions in english this makes it easier for everyone else to find your question and the corresponding answers. Thank you.

**Reply by 136861535** [reply]
The content of the model-server package referenced in the coffee-editor is not exactly the same as the model server code on GitHub. Why?

**Reply by eneufeld** [reply]
Do you mean this package: https://github.com/eclipse-emfcloud/coffee-editor/tree/master/backend/plugins/org.eclipse.emfcloud.coffee.modelserver ?

The modelserver is a framework. It needs to be customized with at least your model. This customization is happening in the linked package.
